test/databases/transaction_database.cpp

```diff
@@ -38,51 +38,502 @@ struct transaction_database_directory_setup_fixture
     {
         test::clear_path(DIRECTORY);
     }
+
+    ~transaction_database_directory_setup_fixture()
+    {
+        test::clear_path(DIRECTORY);
+    }
 };
 
-BOOST_FIXTURE_TEST_SUITE(database_tests, transaction_database_directory_setup_fixture)
+BOOST_FIXTURE_TEST_SUITE(transaction_database_tests, transaction_database_directory_setup_fixture)
+
+BOOST_AUTO_TEST_CASE(transaction_database__store__single_transactions__success)
+{
+    transaction tx1;
+    data_chunk wire_tx1;
+    BOOST_REQUIRE(decode_base16(wire_tx1, TRANSACTION1));
+    BOOST_REQUIRE(tx1.from_data(wire_tx1));
+
+    transaction tx2;
+    data_chunk wire_tx2;
+    BOOST_REQUIRE(decode_base16(wire_tx2, TRANSACTION2));
+    BOOST_REQUIRE(tx2.from_data(wire_tx2));
+
+    const auto path = DIRECTORY "/tx_table";
```

Comment:
I've move this into a file-scoped declaration.

Reply:
Moved, changed var name to file_path to avoid conflicting with boost::filesystem::path. There probably was a better way to resolve it, but I took the easy path - pun intended ;)